Automated saving and cleaned up code/comments

c-randall opened the pull request:
1. Creates a new folder  inside cwd based on user input name and...
- Saves copy of script and cti file
- Saves solution vector including time steps taken
- Creates and saves animation of solution based on 'movie' switch

2. Error code added to ensure that saved files are not overwritten if folder name already exists. User can choose to overwrite or cancel.

3. Erased unnecessary lines and comments to clean up code. Also, cleaned up plots and animation.

c-randall committed:
Automated saving and cleaned up code/comments

1. Creates a new folder inside cwd
2. Saves copy of script and cti file
3. Saves solution vector including time steps taken
4. Creates and saves animation of solution based on 'movie' switch

c-randall committed:
Merge pull request #4 from c-randall/c-randall-patch-4

Automated saving and cleaned up code/comments

c-randall committed:
Update README.md

c-randall committed:
Update README.md

c-randall committed:
Update README.md

c-randall committed:
Fixed Fick's stiffness / added time saving options

1. Fick's Func was stiff due to missing dx and dy terms -> solved
2. New way to copy cti file from local folder added
3. Frames option added to save time in animation generation
4. Maximum outlet density difference added to plot/animation titles

c-randall committed:
Merge pull request #5 from c-randall/c-randall-patch-5

Fixed Fick's stiffness / added time saving options

decaluwe commented:
Hi Corey, one additional item that I don't think has been added to the README yet: along with the diffusion species, the user should supply the moles of electrons produced/consumed per mole of diffusing species.

Thanks.

Commit:
Add PEMFC ability and finalize

Model can now have an outlet flux that is not dependent on O2 and can be utilized for Fuel Cell applications.

Commit:
Merge pull request #6 from c-randall/c-randall-patch-6

Add PEMFC ability and finalize

Commit:
Update README.md

Commit:
Added updated figure to Readme
